### ISSUE TITLE
Question not an issue.

### ISSUE THREAD

**Issue by PatrickDenis** [netfishers-onl/Netshot#87]
I have a question , lets say that i would like to send a cdp ne to all port on all devices to count the number of ap's  base on the OUI,
is this something possible through the diag option ?

**Comment by SCadilhac** [comment]
Yes sure you can run `show cdp nei` in a diagnostic, parse the output to find the number of APs, and store this number as diagnostic result.

**Comment by PatrickDenis** [comment]
Thank you again for the fast response,

So I did follow your example from the wiki how to, but i cant seam to find how to see the output ?
could you give me an example anything easy how to ?

thank you in advance :)

**Comment by SCadilhac** [comment]
![image](https://user-images.githubusercontent.com/14115661/104137805-dbf36900-539f-11eb-8dd9-f0a336002f3f.png)

You can do something like that.

**Comment by PatrickDenis** [comment]
Awesome ! its exactly what i was looking for, time to learn a bite more of JavaScript's.

This lead to a new request :) hehehe only if it is possible :) can this be added to the reports ? that would be amazing ! 
could get rid of Smartnet :)

Thank you again and again :)

**Comment by PatrickDenis** [comment]
Thank You :)
